Core.DomainServices/SystemUsage/ItSystemUsageOverviewReadModelUpdate.cs

```diff
@@ -12,6 +12,22 @@ public void Apply(DomainModel.ItSystemUsage.ItSystemUsage source, ItSystemUsageO
             destination.Name = source.ItSystem.Name;
             destination.ItSystemDisabled = source.ItSystem.Disabled;
             destination.IsActive = source.IsActive;
+            destination.Version = source.Version;
+            destination.LocalCallName = source.LocalCallName;
+
+            PatchParentSystemName(source, destination);
+        }
+
+        private static void PatchParentSystemName(DomainModel.ItSystemUsage.ItSystemUsage source, ItSystemUsageOverviewReadModel destination)
+        {
+            if(source.ItSystem.Parent is null)
```

Comment:
brug null propagation i stedet for alt det der:

`destination.ItSystemParentName = source.ItSystem.Parent?.Name ?? "";`

Reply:
Kan fallback ikke være null ligesom på Version og LocalCallName?... i så fald kan du jo bare

`destination.ItSystemParentName = source.ItSystem.Parent?.Name;` dvs uden fallback til `""`